protob/types.proto

```diff
@@ -506,3 +506,24 @@ message LiskMultisignatureType {
 	optional uint32 life_time = 2;
 	repeated string keys_group = 3;
 }
+
+/**
+ * Structure representing cardano transaction input
+ * @used_in CardanoSignTransacion
+ */
+message CardanoTxInputType {
+	optional bytes tx_hash = 1;				// txHash
```

Comment:
Is it OK if we change the order and names of the fields to match `TxInputType`?

That is 

```
	repeated uint32 address_n = 1;						// BIP-32 path to derive the key from master node
	required bytes prev_hash = 2;						// hash of previous transaction output to spend by this input
	required uint32 prev_index = 3;						// index of previous output to spend
```